rtgui/edit.h

```diff
@@ -152,6 +145,59 @@ class EditBuffer;
  *
  */
 
+
+
+class ObjectMOBuffer
+{
+private:
+
+    // Used to draw the objects where the color correspond to the object's ID, in order to find the correct object when hovering
```

Comment:
This sounds interesting! It is almost certainly out of scope for this change, but isn't this representation relatively wasteful? I.e. what about using an polymorphic `contains` method that checks each geometry item in reverse drawing order if it contains the given coordinate? Should be relatively straight-forward for most geometric primitives like rectangles, circles and polygons and could probably be cached as well. In any case, why not upgrade the surface format instead of drawing twice to a second one if more objects need to be tracked, e.g. using `FORMAT_RGB16_565` and transforming the object identifier into a suitable color?